docs/components/marker.md

```diff
@@ -0,0 +1,34 @@
+# Marker Control
+
+This is a React equivalent of Mapbox's [Marker Control](https://www.mapbox.com/mapbox-gl-js/api/#marker)
```

Comment:
It would be nice to say what it does and not immediately make the user click a link...